roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -13,6 +13,9 @@ storage:
     accesskey: {{ openshift.hosted.registry.storage.s3.accesskey }}
     secretkey: {{ openshift.hosted.registry.storage.s3.secretkey }}
     region: {{ openshift.hosted.registry.storage.s3.region }}
+{%   if openshift.hosted.registry.storage.s3.regionendpoint %}
```

Comment:
Haven't tested but I think this will fail when `regionendpoint` is not set. Checking for the `regionendpoint` key should do the trick.

```
{% if 'regionendpoint' in openshift.hosted.registry.storage.s3 %}
```